tests/Brazil/BrazilBaseTestCase.php

```diff
@@ -0,0 +1,34 @@
+<?php
+/**
+ *  This file is part of the Yasumi package.
+ *
+ *  Copyright (c) 2015 - 2016 AzuyaLabs
+ *
+ *  For the full copyright and license information, please view the LICENSE
+ *  file that was distributed with this source code.
+ *
+ *  @author Dorian Neto <doriansampaioneto@gmail.com>
+ */
+
+namespace Yasumi\tests\Brazil;
+
+use PHPUnit_Framework_TestCase;
+use Yasumi\tests\YasumiBase;
+
+/**
+ * Class BrazilBaseTestCase.
+ */
+abstract class BrazilBaseTestCase extends PHPUnit_Framework_TestCase
+{
+    use YasumiBase;
+
+    /**
+     * Country (name) to be tested
+     */
+    const REGION = 'Brazil';
+
+    /**
+     * Timezone in which this provider has holidays defined
+     */
+    const TIMEZONE = 'America/Fortaleza';
+}
```

Comment:
Please add a constant for the common locale in Brazil. See for example the other test classes (    `const LOCALE = 'nl_NL';`)